tests/fixtures/biattentive_classification_network/feedforward_experiment.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
For some very non-obvious reasons, it's good to have one of these experiment files just named `experiment.json`.  It makes some things easier when we get to making a `Predictor` for this model.